core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -483,8 +483,9 @@ private[spark] class TaskSetManager(
           // a good proxy to task serialization time.
           // val timeTaken = clock.getTime() - startTime
           val taskName = s"task ${info.id} in stage ${taskSet.id}"
-          logInfo("Starting %s (TID %d, %s, %s, %d bytes)".format(
-              taskName, taskId, host, taskLocality, serializedTask.limit))
+          logInfo("Starting %s (TID %d, %s, %d, %s, %d bytes)".format(
```

Comment:
The `s"..."` didn't exist before Spark 2.10, so I think that's why the old style is still used in the code. There's no great need to change all that. I think the interpolated style is clearer, and I tend to think that we should match surrounding code style in issues like this. Since interpolation is used in the line above, it seems right to use it here. I agree it's a tiny issue either way.